docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,190 @@
+iOS:: Uses the iOS link:https://developer.xamarin.com/api/field/ObjCRuntime.Runtime.Arch/[Arch] class
+====
+
+==== Device Lock Detection (SCREEN_LOCK_ENABLED)
+
+To detect if a device has a lock screen set (with pin, fingerprint, pattern) the `SecurityChecks#SCREEN_LOCK_ENABLED` function can be used.
+
+[NOTE]
+====
+The check uses different approaches on Android and iOS to detect if the app is running into an emulator
+[horizontal]
+ANDROID:: Uses the Android's link:https://developer.xamarin.com/api/type/Android.App.KeyguardManager[KeyguardManager] class.
+iOS:: Uses the iOS link:https://developer.xamarin.com/api/type/MonoTouch.LocalAuthentication.LAContext/[LAContext] class
+====
+
+==== App Data Backup Detection (ALLOW_BACKUP_DISABLED)
+
+IMPORTANT: This check is available for Android only
+
+To detect whether the application’s data is configured to be synchronized across devices the `SecurityChecks#ALLOW_BACKUP_DISABLED` function can be used.
+The link:https://developer.android.com/guide/topics/manifest/application-element[allowBackup] flag determines whether to allow the application to
+participate in the backup and restore infrastructure, which might be interesting to avoid depending on your app’s needs.
+
+==== Device Encryption Detection (HAS_ENCRYPTION_ENABLED)
+IMPORTANT: This check is available for Android only
+
+To detect whether the devices filesystem is encrypted the `SecurityChecks#HAS_ENCRYPTION_ENABLED` function can be used.
+This function uses Android’s link:https://developer.xamarin.com/api/type/Android.App.Admin.DevicePolicyManager/[DevicePolicyManager] class.
+
+=== Custom Self Defence Checks
+
+These are self defence checks that you can define to be used by the SDK. Custom self defence checks must implement the link:javadoc[`ISecurityCheck`] interface.
+
+[source, c#]
+----
+class CustomSecurityCheck : ISecurityCheck
+{
+	public string GetName()
+	{
+		return "Custom check";
+	}
+
+	public string GetId()
+	{
+		return typeof(CustomSecurityCheck).FullName;
+	}
+
+	public SecurityCheckResult Check()
+	{
+		// In this example, the check always passes...
+		return new SecurityCheckResult(this, true);
+	}
+}
+----
+
+=== Invoking a Single Self Defence Check
+The `SecurityService#Check` method can be used to run a single self defence check.  It expects either `SecurityChecks` (see <<Provided Self Defence Checks
+>>)  or `ISecurityCheck` (see <<Custom Self Defence Checks
+>>) as a parameter and returns `SecurityCheckResult`.
+
+[source, c#]
+----
+// a provided self defence check
+SecurityCheckResult result = securityService.Check(SecurityChecks.<check_type>);
+
+// using the custom self defence check defined previously
+ISecurityCheck customSecurityCheck = new CustomSecurityCheck();
+SecurityCheckResult result = securityService.Check(customSecurityCheck);
+----
+
+=== Invoking Multiple Self Defence Checks
+
+To invoke multiple self defence checks a security check executor, by calling the `SecurityService.GetSyncExecutor`.
```

Comment:
This sentence is kind of hard to understand.  Maybe reword it to something like "To invoke multiple self defence checks a security check executor must be used.  To run multiple self defence checks synchronously,  you'll need to invoke `SecurityService#GetSyncExecutor`.  "